plugins/vpc-shared-eni/network/network.go

```diff
@@ -35,7 +35,9 @@ type Network struct {
 	BridgeIndex         int
 	SharedENI           *eni.ENI
 	ENIIPAddress        *net.IPNet
+	ENIIPv6Address      *net.IPNet
```

Comment:
Same here. Let's make the existing field an array instead of adding IPv6-specific fields. This will also cover multiple IP address use cases.